Formula/arangodb.rb

```diff
@@ -21,16 +21,28 @@ class Arangodb < Formula
     cause "Fails with compile errors"
   end
 
+  # see https://gcc.gnu.org/bugzilla/show_bug.cgi?id=87665
   fails_with :gcc do
     build 820
     cause "Generates incorrect code"
   end
 
+  resource "starter" do
+    url "https://github.com/arangodb-helper/arangodb.git",
+        :revision => "38953b049a38c844ffdbf667e55e60c6cfa22e70"
+  end
+
   needs :cxx11
 
   def install
     ENV.cxx11
 
+    (buildpath/"starter").install resource("starter")
```

Comment:
Could we do:

```
resource("starter").stage do
  system "make", "NODOCKER=yes", "local"
  bin.install "arangodb"
end
```

Reply:
Also, we need to test that binary in `test do` block.

Reply:
done